include/Synth/Synth.h

```diff
@@ -0,0 +1,118 @@
+#ifndef MPU_SYNTH_H
+#define MPU_SYNTH_H
+
+#include "IMU/MPU6050.h"
+#include "Button/Button.h"
+#include <vector>
+#include <array>
+#include <string>
+#include <mutex>
+#include <atomic>
+#include <thread>
+#include <cmath>
+#include <iostream>
+#include <map>
+
+// STK includes
+#include <stk/Instrmnt.h>
+#include <stk/Rhodey.h>
+#include <stk/Wurley.h>
+#include <stk/Moog.h>
+#include <stk/Clarinet.h>
+#include <stk/RtAudio.h>
+#include <stk/Stk.h>
+
+// Define constants
+#define ACCEL_SCALE 16384.0
+#define G 9.81
+#define ACCEL_XOUT_H 0x3B
+#define ACCEL_YOUT_H 0x3D
+#define ACCEL_ZOUT_H 0x3F
+
+// Scale type definitions
+enum class ScaleType {
+    MAJOR,
+    MINOR,
+    PENTATONIC,
+    BLUES,
+    CHROMATIC
+};
+
+// Instrument type definitions
+enum class InstrumentType {
+    PIANO,
+    RHODEY,
+    WURLEY,
+    MOOG,
+    CLARINET
+};
+
+class MPUSynth {
```

Comment:
MPUSynth should inherit a callback interface defined in MPU6050 class. The MPU6050 class should have a callback struct, just like the one present in Button class. This struct must have a hasEvent method which should be implemented by the MPUSynth class.

Reply:
The new struct within MPU6050 can be called a ControllerInterface.